src/playwrightTest.ts

```diff
@@ -66,6 +66,30 @@ export class PlaywrightTest {
   }
 
   async getPlaywrightInfo(workspaceFolder: string, configFilePath: string): Promise<{ version: number, cli: string } | null> {
+
+    if (isPnPWorkspace(workspaceFolder)) {
+      try {
+        const pwtCoreInfo = await this._runNode([
+          '-e',
+          'const pnpapi = require("pnpapi"); ' +
```

Comment:
Is this the package you are using? https://www.npmjs.com/package/pnpapi

Reply:
No, in yarn-v3 PnP environments `pnpapi` is a built-in module. The package on npm is just a placeholder and not used.  
Details can be found here [https://yarnpkg.com/advanced/pnpapi#requirepnpapi](https://yarnpkg.com/advanced/pnpapi#requirepnpapi)